build.xml

```diff
@@ -93,4 +93,12 @@
         </exec>
     </target>
 
+    <!-- This is known to fail but gives an idea of what the hot issues are. -->
+    <target name="run.test.for.testDefinitelyTyped">
```

Comment:
I'm familiar with that dialog, but I don't know kind of execution it should be with what options.  I don't know what Plugins I need to install.